bc_obps/compliance/tests/service/bc_carbon_registry/test_apply_compliance_units_service.py

```diff
@@ -542,6 +546,8 @@ def test_apply_compliance_units_filters_zero_quantities(
 
         mock_bccr_service.client.transfer_compliance_units.return_value = {"success": True}
 
+        mock_bccr_service.validate_holding_account_ownership.return_value = True
```

Comment:
It would be a good test to assert that `validate_holding_account_ownership` is called after applying units.